### DISCUSSION TITLE
Valgfri font size på Expandbutton

### DISCUSSION THREAD

**Discussion by ivnai** [fremtind/jokul#3389] [Annet]
Hallo! Er det ikke god idee, bruke ExpandButton til både, overskrift og åpne/skjul funksjon?
Her på Flyt ønsker vi oss knappen hvor man kan endre fontstil til en annen fra Jøkul.
Hva er tankene?
Er det mulig å utføre endringene? 
<img width="985" alt="Screenshot 2023-02-09 at 09 45 18" src="https://user-images.githubusercontent.com/93194579/217763524-99dd738b-73dc-4b84-8c6f-fa3a188e3bb6.png">

**Comment by wkillerud** [comment]
Ja, jeg tror det skal gå bra. Kan se for meg at det er nyttig med den muligheten. Laget et issue: 

- https://github.com/fremtind/jokul/issues/3392

Er det sånn at en bruker ofte leter etter noe spesifikt i ProLife, Paris og NICE? Kan tabs være en alternativ måte å dele innhold på?

**Comment by ivnai** [comment]
Takk @wkillerud ! Fordi horisontale kortene er allerede expandable (på flere nivåer), så ønsker vi å bruke det samme mønstre på hele side; ren expand 💅🏼  Bruker kan både, lete etter noe spesifikk eller scanne alle.
